src/main/java/bio/terra/service/tabulardata/google/BigQueryProject.java

```diff
@@ -164,12 +165,12 @@ public void addDatasetAcls(String datasetId, List<Acl> acls) {
     updateDatasetAcls(dataset, newAcls);
   }
 
-  public void removeDatasetAcls(String datasetId, List<Acl> acls) {
+  public void removeDatasetAcls(String datasetId, List<Acl> acls) throws InterruptedException {
     Dataset dataset = bigQuery.getDataset(datasetId);
     if (dataset != null) { // can be null if create dataset step failed before it was created
-      Set<Acl> datasetAcls = new HashSet(dataset.getAcl());
+      Set<Acl> datasetAcls = new HashSet<>(dataset.getAcl());
       datasetAcls.removeAll(acls);
-      updateDatasetAcls(dataset, new ArrayList(datasetAcls));
+      updateDatasetAcls(dataset, new ArrayList<>(datasetAcls));
```

Comment:
Same: `List.of`

Reply:
Same as above

Reply:
To wrap a different type of collection in a `List` you use `copyOf()`, not `List.of()`. `updateDatasetAcls()` doesn't modify its argument so using the `copyOf()` result should be fine.
```suggestion
      updateDatasetAcls(dataset, List.copyOf(datasetAcls));
```